inc/Api/Api.php

```diff
@@ -58,9 +58,19 @@ public function register_api_information() {
 				} elseif ( $sanitized['password'] === '' ) {
 					$response = array( 'error' => 'Please enter password!' );
 				} else {
+					// Get subdomain if entered as full url.
+					if ( filter_var( $sanitized['subdomain'], FILTER_VALIDATE_URL ) ) {
+						// Get host adress.
+						$url = wp_parse_url( $sanitized['subdomain']);
```

Comment:
Formatting.

Reply:
Fixed in https://github.com/sendsmaily/smaily-woocommerce-plugin/pull/27/commits/279bdd8f57579fcb459659b33854b32ac2f862ee